### PR TITLE
ID translation for access cluster

### PR DESCRIPTION
**Pull request by deryrahman**
This PR is part of #20111 , translating `/docs/tasks/access-application-cluster/access-cluster/` into bahasa

/language id

### CODE REVIEW

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+    - berjalan di desktop pengguna atau di pod
+    - *proxy* dari alamat localhost ke apiserver Kubernetes
+    - klien ke *proxy* menggunakan HTTP
+    - *proxy* ke apiserver menggunakan HTTPS
+    - pencarian apiserver
+    - menambahkan *header* autensikasi
+
+1.  [*Proxy* apiserver](#menemukan-servis-builtin):
+
+    - merupakan pertahanan yang dibangun ke dalam apiserver
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:312]:
```suggestion
    - merupakan sebuah _bastion_ yang dibangun ke dalam apiserver
```

"Basiton" is a term -> https://en.wikipedia.org/wiki/Bastion_host

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+    - menghubungkan pengguna di luar klaster ke IP klaster yang mungkin tidak dapat dijangkau
+    - berjalan dalam proses apiserver
+    - klien ke *proxy* menggunakan (atau http jika apiserver dikonfigurasi)
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:315]:
```suggestion
    - dari klien ke proksi menggunakan HTTPS (atau HTTP jika apiserver dikonfigurasi)
```

HTTPS-nya ketinggalan

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+    - *proxy* ke target dapat menggunakan HTTP atau HTTPS seperti yang dipilih oleh *proxy* menggunakan informasi yang tersedia
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:316]:
```suggestion
    - dari proksi ke target dapat menggunakan HTTP atau HTTPS seperti yang dipilih oleh proksi menggunakan informasi yang tersedia
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+    - dapat digunakan untuk menjangkau Node, Pod, atau Servis
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:317]:
```suggestion
    - dapat digunakan untuk menjangkau Node, Pod, atau Service
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+    - melakukan *load balancing* saat digunakan untuk menjangkau Servis
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:318]:
```suggestion
    - melakukan _load balancing_ saat digunakan untuk menjangkau sebuah Service
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+
+1.  [*Proxy* kube](/docs/concepts/services-networking/service/#ips-and-vips):
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:320]:
```suggestion
1.  [kube-proxy](/docs/concepts/services-networking/service/#ips-and-vips):
```

Kube-proxy is the name of a Kubernetes component. Let's use it as is.

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 7]
```diff
@@ -0,0 +1,343 @@
...
+
+    - berjalan di setiap node
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:322]:
```suggestion
    - berjalan di setiap Node
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 8]
```diff
@@ -0,0 +1,343 @@
...
+    - *proxy* UDP dan TCP
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:323]:
```suggestion
    - memproksi UDP dan TCP
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 9]
```diff
@@ -0,0 +1,343 @@
...
+    - tidak mengerti HTTP
+    - menyediakan *load balancing*
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:325]:
```suggestion
    - menyediakan _load balancing_
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 10]
```diff
@@ -0,0 +1,343 @@
...
+    - hanya digunakan untuk menjangkau servis
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:326]:
```suggestion
    - hanya digunakan untuk menjangkau Service
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 11]
```diff
@@ -0,0 +1,343 @@
...
+
+1.  *Proxy*/*Load-balancer* di depan apiserver:
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:328]:
```suggestion
1.  Sebuah Proksi/_Load-balancer_ di depan apiserver:
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 12]
```diff
@@ -0,0 +1,343 @@
...
+
+    - keberadaan dan implementasi bervariasi dari satu klaster ke klaster lainnya (mis. nginx)
+    - terletak di antara semua klien dan satu atau lebih penyedia servis
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:331]:
```suggestion
    - terletak di antara semua klien dan satu atau lebih apiserver
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 13]
```diff
@@ -0,0 +1,343 @@
...
+    - bertindak sebagai *load balancer* jika terdapat beberapa apiserver.
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:332]:
```suggestion
    - bertindak sebagai _load balancer_ jika terdapat beberapa apiserver.
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 14]
```diff
@@ -0,0 +1,343 @@
...
+
+1.  *Cloud Load Balancer* pada servis eksternal:
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:334]:
```suggestion
1.  _Cloud Load Balancer_ pada Service eksternal:
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 15]
```diff
@@ -0,0 +1,343 @@
...
+
+    - disediakan oleh beberapa penyedia cloud (mis. AWS ELB, Google Cloud Load Balancer)
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:336]:
```suggestion
    - disediakan oleh beberapa penyedia layanan _cloud_ (mis. AWS ELB, Google Cloud Load Balancer)
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 16]
```diff
@@ -0,0 +1,343 @@
...
+    - dibuat secara otomatis ketika servis Kubernetes memiliki tipe `LoadBalancer`
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:337]:
```suggestion
    - dibuat secara otomatis ketika Service Kubernetes memiliki tipe `LoadBalancer`
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 17]
```diff
@@ -0,0 +1,343 @@
...
+    - hanya menggunakan UDP/TCP
+    - implementasi bervariasi berdasarkan penyedia cloud.
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:339]:
```suggestion
    - implementasi bervariasi berdasarkan penyedia layanan _cloud_.
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 18]
```diff
@@ -0,0 +1,343 @@
...
+
+Pengguna Kubernetes biasanya tidak perlu khawatir tentang apa pun selain dua jenis pertama. Admin klaster biasanya akan memastikan bahwa tipe yang terakhir diatur dengan benar.
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:341]:
```suggestion
Pengguna Kubernetes biasanya tidak perlu khawatir tentang apa pun selain dua jenis pertama. Admin klaster biasanya akan memastikan bahwa tipe yang terakhir telah diatur dengan benar.
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 19]
```diff
@@ -0,0 +1,343 @@
...
+
+Kubectl menangani pencarian dan autentikasi ke apiserver. 
+Jika kamu ingin secara langsung mengakses REST API dengan klien http seperti curl atau wget, atau peramban, ada beberapa cara untuk pencarian dan autentikasi:
+
+  - Jalankan kubectl dalam mode *proxy*.
+    - Pendekatan yang disarankan.
+    - Menggunakan lokasi apiserver yang tersimpan.
+    - Melakukan verifikasi identitas apiserver menggunakan sertifikat yang ditandatangani sendiri. Tidak memungkinkan adanya MITM.
+    - Melakukan autentikasi ke apiserver.
+    - Di masa depan, dimungkinkan dapat melakukan *load-balancing* dan *failover* yang cerdas dari sisi klien.
```
Comment (phanama) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:43]:
```suggestion
    - Di masa depan, dimungkinkan dapat melakukan _load-balancing_ dan _failover_ yang cerdas dari sisi klien.
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 20]
```diff
@@ -0,0 +1,343 @@
...
+Kubernetes master is running at https://104.197.5.247
+elasticsearch-logging is running at https://104.197.5.247/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy
+kibana-logging is running at https://104.197.5.247/api/v1/namespaces/kube-system/services/kibana-logging/proxy
+kube-dns is running at https://104.197.5.247/api/v1/namespaces/kube-system/services/kube-dns/proxy
+grafana is running at https://104.197.5.247/api/v1/namespaces/kube-system/services/monitoring-grafana/proxy
+heapster is running at https://104.197.5.247/api/v1/namespaces/kube-system/services/monitoring-heapster/proxy
+```
+
+Ini menunjukkan URL _proxy-verb_ untuk mengakses setiap Service.
+Misalnya, klaster ini mempunyai pencatatan log pada level klaster (_cluster-level logging) yang aktif (menggunakan Elasticsearch), yang dapat dicapai di `https://104.197.5.247/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/` jika kredensial yang sesuai diberikan.
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:243]: missing underscore
```suggestion
Misalnya, klaster ini mempunyai pencatatan log pada level klaster (_cluster-level logging_) yang aktif (menggunakan Elasticsearch), yang dapat dicapai di `https://104.197.5.247/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/` jika kredensial yang sesuai diberikan.
```

File: content/id/docs/tasks/access-application-cluster/access-cluster.md [hunk 21]
```diff
@@ -0,0 +1,343 @@
...
+
+## Begitu Banyaknya Proksi
+
+Ada beberapa proksi berbeda yang mungkin kamu temui saat menggunakan Kubernetes:
+
+1.  [Proksi kubectl](#mengakses-rest-api-secara-langsung):
+
+    - berjalan di desktop pengguna atau di Pod
+    - memproksi dari sebuah alamat localhost ke apiserver Kubernetes
+    -  dari klien ke proksi menggunakan HTTP
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/access-cluster.md:305]: nit: extra space not required
```suggestion
    - dari klien ke proksi menggunakan HTTP
```

**Comment by girikuncoro** [comment]
Thanks again!

/lgtm
/approve

**Pull request merged**